Packs/ArcherRSA/ReleaseNotes/1_1_1.md

```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### RSA Archer - Deprecated
+- We recommend using RSA Archer v2 instead.
```

Comment:
```suggestion
##### RSA Archer
Deprecated. We recommend using RSA Archer v2 instead.

```

Reply:
i changed it to:

```

#### Integrations
##### RSA Archer (Deprecated)
Deprecated. We recommend using RSA Archer v2 instead.

```
after gets this error in validate:
`Packs/ArcherRSA/ReleaseNotes/1_1_1.md: [RN107] - No release note entry was found for the integration "RSA Archer (Deprecated)" in the ArcherRSA pack.`